set.mm

```diff
@@ -46697,26 +46709,27 @@ although the definition does not require it (see ~ dfid2 for a case
     $( Change first bound variable in an ordered-pair class abstraction, using
        explicit substitution.  (Contributed by NM, 6-Oct-2004.)  (Revised by
        Mario Carneiro, 14-Oct-2016.)  Add disjoint variable condition to avoid
-       ~ ax-13 .  (Revised by Gino Giotto, 17-Jan-2024.) $)
+       ~ ax-13 .  See ~ cbvopab1g for a less restrictive version requiring more
+       axioms.  (Revised by Gino Giotto, 17-Jan-2024.) $)
     cbvopab1 $p |- { <. x , y >. | ph } = { <. z , y >. | ps } $=
       ( vw vv cv cop wceq wa wex cab copab nfv nfan nfex wsb nfs1v opeq1 eqeq2d
       sbequ12 anbi12d exbidv cbvexv1 nfsbv sbhypf bitri abbii df-opab 3eqtr4i )
       IKZCKZDKZLZMZANZDOZCOZIPUOEKZUQLZMZBNZDOZEOZIPACDQBEDQVBVHIVBUOJKZUQLZMZA
       CJUAZNZDOZJOVHVAVNCJVAJRVMCDVKVLCVKCRACJUBSTUPVIMZUTVMDVOUSVKAVLVOURVJUOU
       PVIUQUCUDACJUEUFUGUHVNVGJEVMEDVKVLEVKERACJEFUISTVGJRVIVCMZVMVFDVPVKVEVLBV
       PVJVDUOVIVCUQUCUDABCJVCGHUJUFUGUHUKULACDIUMBEDIUMUN $.
+    $( $j usage 'cbvopab1' avoids 'ax-13'; $)
   $}
 
   ${
     $d v w x y $.  $d v w y z $.  $d v w ph $.  $d v w ps $.
-    cbvopab1ALT.1 $e |- F/ z ph $.
-    cbvopab1ALT.2 $e |- F/ x ps $.
-    cbvopab1ALT.3 $e |- ( x = z -> ( ph <-> ps ) ) $.
-    $( Alternate version of ~ cbvopab1 , with less dv conditions, but requiring
-       ~ ax-13 .  (Contributed by NM, 6-Oct-2004.)  (Revised by Mario Carneiro,
-       14-Oct-2016.)  (Proof modification is discouraged.)
-       (New usage is discouraged.) $)
-    cbvopab1ALT $p |- { <. x , y >. | ph } = { <. z , y >. | ps } $=
+    cbvopab1g.1 $e |- F/ z ph $.
+    cbvopab1g.2 $e |- F/ x ps $.
+    cbvopab1g.3 $e |- ( x = z -> ( ph <-> ps ) ) $.
+    $( Change first bound variable in an ordered-pair class abstraction, using
```

Comment:
Both, the *g and the * variant should point to each other.  "See ~ cbvopab1 with more dv conditions, but without ~ ax-13 ."

Reply:
Done.

Reply:
Thanks for the careful PR and edits.